### PR TITLE
MBL-12888 - if assignment details description is empty, show "No Content"

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-12888
affects: Student
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Student

[MBL-12888](https://instructure.atlassian.net/browse/MBL-12888)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Core** | 95.34% | 95.34% | 0%
**Student** | 91.08% | 91.05% | 0.03%
**Teacher** | 100% | 100% | 0%
**React Native** | 92.79% | 92.79% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Student/Student/Assignments/AssignmentDetails/AssignmentDetailsViewController.swift [hunk 1]
```diff
@@ -228,7 +228,7 @@ class AssignmentDetailsViewController: UIViewController, AssignmentDetailsViewPr
         descriptionHeadingLabel?.text = quiz == nil
             ? NSLocalizedString("Description", bundle: .student, comment: "")
             : NSLocalizedString("Instructions", bundle: .student, comment: "")
-        descriptionView?.loadHTMLString(assignment.descriptionHTML, baseURL: baseURL)
+        descriptionView?.loadHTMLString(presenter?.assignmentDescription() ?? "", baseURL: baseURL)
```
Comment (shichongrui) [inline Student/Student/Assignments/AssignmentDetails/AssignmentDetailsViewController.swift]: Why is the `?? ""` necessary? `assignmentDescription` should always return some string right?

Comment (bluwave) [reply Student/Student/Assignments/AssignmentDetails/AssignmentDetailsViewController.swift]: but the presenter is optional, i could do a guard for the presenter but ...meh...

Comment (shichongrui) [reply Student/Student/Assignments/AssignmentDetails/AssignmentDetailsViewController.swift]: Ohh I missed that. Gotcha.

**Comment by shichongrui** [comment]
Code looks good. QA'ing

**Pull request merged**